cli/pcluster/cfnconfig.py

```diff
@@ -490,8 +490,6 @@ def __init_cluster_options():
             s3_read_resource=('S3ReadResource', None),
             s3_read_write_resource=('S3ReadWriteResource', None),
             tenancy=('Tenancy', None),
-            ephemeral_kms_key_id=('EphemeralKMSKeyId', None),
```

Comment:
I would like to have this in another PR together with the template changes

Reply:
Will do. I thought that splitting the changes in separate commits was good enough but probably it's worth having a separate PR for this.